### PR TITLE
RHOAIENG-10128 - Update Readme for the manifest directory

### PR DESCRIPTION
**Pull request by MarianMacik**
**JIRA:** [RHOAIENG-10128](https://issues.redhat.com/browse/RHOAIENG-10128)
## Description

This splits the original README.md to `manifests/README.md` (how to run pipelines) and `pipelines/README.md` (model description).

## How Has This Been Tested?
N/A as this is just about the documentation.

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Review by jackdelahunt — APPROVED** [review]
/lgtm

**Comment by jackdelahunt** [comment]
Just a side question should we be calling it the `pipelines` folder anymore. Seems like it is a combonation of info about models and the models and example pipeline runs. Its not big deal just wanted to see what you thought

File: README.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ The proof of concept is built to provide two main workflows as shown in the diag
 
 ### Workflow 1: Building an Inference Application Container Image
 
-In [pipelines/README.md](pipelines/README.md) we show how to take the trained models,
+In [manifests/README.md](manifests/README.md) we show how to take the trained models,
 store them in a S3 bucket,
```
Comment (biswassri) [inline README.md:98]: Small nit,
```suggestion
stored in a S3 bucket or Git repository,
```

**Comment by biswassri** [comment]
@jackdelahunt  I agree, maybe we should rename it cause it might be confusing for anyone looking at this repo. 

Also, I was thinking we should add more information in the pipeline readme about the tekton package, that it contains the same pipeline runs and containerfiles. Basically more information about the contents of the pipeline folder. WDYT?

**Comment by jackdelahunt** [comment]
@biswassri 

I am always pro giving users more info but I am not sure what you mean

> about the tekton package
I am not sure what tekton package this is about

>  that it contains the same pipeline runs and containerfiles
so instead of docs how to run the pipeline also mentioning the examples other then just the guide and what else we have there

**Review by biswassri — APPROVED** [review]
/lgtm 
Thanks @MarianMacik

**Pull request merged**